### ISSUE TITLE
Add issues template

### ISSUE THREAD

**Issue by BenWiederhake** [majn/telegram-purple#241]
Use the new feature that allows us to [tell the user what to report](https://github.com/blog/2111-issue-and-pull-request-templates).

This should help greatly in reducing the amount of roundtrips. Keep in mind that this MUST NOT exceed 5 lines, since the mental user must be kept as low as possible.

Do we want to suggest a reporting template for tgl, too?

**Comment by BenWiederhake** [comment]
Quick stats about what could have prevented by a template:
- 4x issue lacks log
- 2x issue lacks concrete error message / demonstration
- 1x issue already fixed in pipeline
- 1x issue already fixed in earlier version
- 1x issue about missing channel/supergroup support
- 1x submodule usability

Basing on this I suggest this as a template:

> Before you submit, please check if the issue already has been fixed in a newer version.
> Ideally, you build it [from the source](https://github.com/majn/telegram-purple#building-from-source).
> Please note that super-groups (groups with 200+ members) and channels are not completely implemented yet. Feedback is appreciated, but we already know that it mostly doesn't work yet.
> 
> Often it's important to know what sequence of actions led up to the crash/error/warning. Try collect a debug log during the crash. For pidgin, the command `pidgin -d | tee debug.log` would do that. You may want to edit out personal information before uploading this file. If you're super-awesome, you can even [generate a backtrace](https://developer.pidgin.im/wiki/GetABacktrace) for us.
> 
> If it's not really a crash/error, please describe what behavior you would have expected. For example: "It doesn't work!" is a bad description. A better description would be "If I click on the red button, then the program exits, but I expected a puppy appearing!"
> 
> Thanks for your report!

**Comment by BenWiederhake** [comment]
Updated template, if 1.3.0 ever gets ready:

**Comment by BenWiederhake** [comment]
If anyone feels like implementing this enhancement, it might be a better idea to direct your energy to [tdlib-purple](https://github.com/ars3niy/tdlib-purple). For now, there seems to be nobody who implements any enhancements for telegram-purple, so I'm closing all "enhancement" issues.

tl;dr: :'(
